src/components/FormLogin.vue

```diff
@@ -45,10 +53,19 @@ export default defineComponent({
     });
 
     const isPassword = ref(true);
-    const formState = ref<'login' | 'password-reset'>('login');
+    const formState = ref<'login' | 'password-reset' | 'reset-finished'>(
+      'login',
+    );
+
+    const backgroundColor = rideToWorkByBikeConfig.colorWhiteOpacity;
 
     const isValid = (val: string): boolean => val?.length > 0;
 
+    const isEmail = (value: string): boolean => {
+      const regex = /^[a-zA-Z0-9+_.-]+@[a-zA-Z0-9.-]+\.[a-zA-z]{2,3}$/;
```

Comment:
[source](https://en.wikipedia.org/wiki/Email_address): _dot ., provided that it is not the first or last character and provided also that it does not appear consecutively (e.g., John..Doe@example.com is not allowed)_

Reply:
Edited based on linked requirements ([regex source](https://uibakery.io/regex-library/email)). Added email validation test.
(As discussed, certain valid but very unusual addresses are currently not supported in favour of a more concise regex rule.)